data/json/player_activities.json

```diff
@@ -136,51 +156,59 @@
   {
     "id": "ACT_FIRSTAID",
     "type": "activity_type",
+    "activity_level": "LIGHT_EXERCISE",
     "stop_phrase": "Stop using first aid?",
     "based_on": "speed"
   },
   {
     "id": "ACT_FISH",
     "type": "activity_type",
+    "activity_level": "LIGHT_EXERCISE",
     "stop_phrase": "Stop fishing?",
     "rooted": true,
     "based_on": "time"
   },
   {
     "id": "ACT_PICKAXE",
     "type": "activity_type",
+    "activity_level": "EXTRA_EXERCISE",
     "stop_phrase": "Stop mining?",
     "based_on": "speed"
   },
   {
     "id": "ACT_BURROW",
     "type": "activity_type",
+    "activity_level": "NO_EXERCISE",
```

Comment:
NO_EXERCISE for burrowing?

Probably should be EXTRA_EXERCISE or ACTIVE_EXERCISE.